### PR TITLE
chore: upgrade calldata

### PR DESCRIPTION
**Pull request by kelemeno**
## Checklist

- [ ] PR title corresponds to the body of PR (we generate changelog entries from PRs).
- [ ] Tests for the changes have been added / updated.
- [ ] Documentation comments have been added / updated.
- [ ] Code has been formatted via `zk fmt` and `zk lint`.
- [ ] Spellcheck has been run via `zk spellcheck`.

### CODE REVIEW

File: etc/env/base/contracts.toml [hunk 1]
```diff
@@ -55,7 +55,7 @@ MAX_NUMBER_OF_HYPERCHAINS = 100
 L1_SHARED_BRIDGE_PROXY_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
 L2_SHARED_BRIDGE_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
 L2_SHARED_BRIDGE_IMPL_ADDR = "0xFC073319977e314F251EAE6ae6bE76B0B3BAeeCF"
-FRI_RECURSION_LEAF_LEVEL_VK_HASH = "0xcc4ac1853353538a166f5c2dde2c24e7e6c461dce8e3dc47d81e9139e1719456"
+FRI_RECURSION_LEAF_LEVEL_VK_HASH = "0xf9664f4324c1400fa5c3822d667f30e873f53f1b8033180cd15fe41c1e2355c6"
```
Comment (StanislavBreadless) [inline etc/env/base/contracts.toml:58]: please double check with @EmilLuta that it is fine to merge to main now

File: etc/env/configs/mainnet.toml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+CONTRACTS_CREATE2_FACTORY_ADDR="0xce0042b868300000d44a59004da54a005ffdcf9f"
+CONTRACTS_DIAMOND_PROXY_ADDR="0x32400084c286cf3e17e7b677ea9583e60a000324"
+CONTRACTS_ERA_DIAMOND_PROXY_ADDR="0x32400084c286cf3e17e7b677ea9583e60a000324"
+CONTRACTS_L1_WETH_TOKEN_ADDR="0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2"
+CONTRACTS_L2_ERC20_BRIDGE_ADDR="0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102"
+CONTRACTS_L2_SHARED_BRIDGE_ADDR="0x11f943b2c77b743AB90f4A0Ae7d5A4e7FCA3E102"
+CONTRACTS_L1_ERC20_BRIDGE_PROXY_ADDR="0x57891966931Eb4Bb6FB81430E6cE0A03AAbDe063"
+CONTRACTS_BASE_TOKEN_ADDR="0x0000000000000000000000000000000000000001"
+CONTRACTS_GOVERNANCE_ADDR="0x0b622A2061EaccAE1c664eBC3E868b8438e03F61"
+# CONTRACTS_VALIDATOR_TIMELOCK_EXECUTION_DELAY
```
Comment (StanislavBreadless) [inline etc/env/configs/mainnet.toml:21]: please set to real value for mainnet

File: etc/env/configs/stage-proofs.toml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+CONTRACTS_CREATE2_FACTORY_ADDR="0xce0042b868300000d44a59004da54a005ffdcf9f"
+CONTRACTS_DIAMOND_PROXY_ADDR="0x5BBdEDe0F0bAc61AA64068b60379fe32ecc0F96C"
+CONTRACTS_ERA_DIAMOND_PROXY_ADDR="0x5BBdEDe0F0bAc61AA64068b60379fe32ecc0F96C"
+CONTRACTS_L1_WETH_TOKEN_ADDR="0x7b79995e5f793A07Bc00c21412e50Ecae098E7f9"
+CONTRACTS_L2_ERC20_BRIDGE_ADDR="0x5978EE0398104a68De718c70cB60a4afdeD07EEE"
+CONTRACTS_L2_SHARED_BRIDGE_ADDR="0x5978EE0398104a68De718c70cB60a4afdeD07EEE"
+CONTRACTS_L1_ERC20_BRIDGE_PROXY_ADDR="0x26d60F0ac5dd7a8DBE98DCf20c0F4b057Ed62775"
+CONTRACTS_BASE_TOKEN_ADDR="0x0000000000000000000000000000000000000001"
+CONTRACTS_GOVERNANCE_ADDR="0xbF4B985eACb623aAFd0B90D9F8C794fa8585edE9"
+# CONTRACTS_VALIDATOR_TIMELOCK_EXECUTION_DELAY
```
Comment (StanislavBreadless) [inline etc/env/configs/stage-proofs.toml:21]: remove/set real value

File: etc/env/configs/testnet.toml [hunk 4]
```diff
@@ -0,0 +1,30 @@
+__imports__ = ["base", "l1-inits/testnet.env", "l2-inits/testnet.init.env" ]
+ZKSYNC_DEBUG_LOGS=true
+CHAIN_ETH_ZKSYNC_NETWORK_ID=300
+CONTRACTS_ERA_CHAIN_ID="300"
+ETH_CLIENT_WEB3_URL= ""
+# MISC_ETHERSCAN_API_KEY = ""
```
Comment (StanislavBreadless) [inline etc/env/configs/testnet.toml:6]: above we use "" for fields that are to be set by the user explicitly, please do the same ehre

File: etc/env/l1-inits/stage-proofs.env [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+CONTRACTS_BRIDGEHUB_PROXY_ADDR=0x7BDF7970F17278a6Ff75Fdbc671E870b0728ae41
+CONTRACTS_ADMIN_FACET_ADDR=0xE698A6Fb588A7B4f5b4C7478FCeC51aB8f869B36
+CONTRACTS_MAILBOX_FACET_ADDR=0x3aA2A5f021E546f4fe989Fc4b428099D1FA853F5
+CONTRACTS_GETTERS_FACET_ADDR=0x22588e7cac6770e43FB99961Db70c608c45D9924
+CONTRACTS_DIAMOND_INIT_ADDR=0xaee9C9FfDcDcB2165ab06E07D32dc7B46379aA3e
+CONTRACTS_STATE_TRANSITION_IMPL_ADDR=0x99D662d6eAf20bc0aAD185D58BdF945abfc8eDa2
+CONTRACTS_STATE_TRANSITION_PROXY_ADDR=0x925Dd0BC14552b0b261CA8A23ad26df9C6f2C8bA
+CONTRACTS_L1_SHARED_BRIDGE_IMPL_ADDR=0xAADA1d8Ec8Bc342a642fAEC52F6b92A2ea4411F3
+CONTRACTS_L1_SHARED_BRIDGE_PROXY_ADDR=0xc488a65b400769295f8C4b762AdCB3E6a036220b
+CONTRACTS_L1_ERC20_BRIDGE_IMPL_ADDR=0x3aF396F034F64A3DC7A1c5F4295d6a827332f100
```
Comment (StanislavBreadless) [inline etc/env/l1-inits/stage-proofs.env:18]: nit newline

File: etc/env/l2-inits/stage.init.env [hunk 6]
```diff
@@ -0,0 +1 @@
+CONTRACTS_L2_SHARED_BRIDGE_IMPL_ADDR=0xce0a8c005a73e35d95cec41a9e8b75668470fb8f
```
Comment (StanislavBreadless) [inline etc/env/l2-inits/stage.init.env:1]: same here

File: etc/upgrades/1711451944-hyperchain-upgrade/mainnet/otherUgrades.json [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
...0000000000000000000000000000000000000000000000000000000000000000000000d7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000847a1d8d3a000000000000000000000000cfa04b9bf3c346b2ac9d3121c1593ba8dd30bcd500000000000000000000000057891966931eb4bb6fb81430e6ce0a03aabde063000000000000000000000000000000000000000000000000000000000000014400000000000000000000000000000000000000000000000000000000000493e000000000000000000000000000000000000000000000000000000000000000000000000000000000d7f9f54194c633f36ccd5f3da84ad4a1c38cb2cb0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000006000000000000000000000000000000000000000000000000000000000000000847a1d8d3a000000000000000000000000af5191b0de278c7286d6c7cc6ab6bb8a73ba2cd600000000000000000000000057891966931eb4bb6fb81430e6ce0a03aabde063000000000000000000000000000000000000000000000000000000000000014400000000000000000000000000000000000000000000000000000000000493e000000000000000000000000000000000000000000000000000000000",
```
Comment (StanislavBreadless) [inline etc/upgrades/1711451944-hyperchain-upgrade/mainnet/otherUgrades.json:46]: not sure why, but it shows that the tabulation is inconsistent in this file

Comment (kelemeno) [reply etc/upgrades/1711451944-hyperchain-upgrade/mainnet/otherUgrades.json:46]: I though fmt would catch it

File: etc/env/base/contracts.toml (hunk 1 above)
Comment (kelemeno) [reply etc/env/base/contracts.toml:58]: I can also just revert

File: etc/upgrades/1711451944-hyperchain-upgrade/localhost/crypto.json [hunk 8]
```diff
@@ -1,7 +1,7 @@
 {
   "verifier": {
-    "address": "0x3b504cD804b65735b1346fe4aFEfc4B3478518b1",
-    "txHash": "0x1f39124e7dded035a620893118e70a3554a798ed3ca8f410b7e0c438f4e976f5"
+    "address": "0x798A686b4b2Fd5a827eeCAfFd9356acb7C7b361e",
```
Comment (StanislavBreadless) [inline etc/upgrades/1711451944-hyperchain-upgrade/localhost/crypto.json:3]: please delete this folder altogether

File: etc/env/configs/stage.toml [hunk 9]
```diff
@@ -0,0 +1,29 @@
+__imports__ = ["base", "l1-inits/stage.env", "l2-inits/stage.init.env" ]
+ZKSYNC_DEBUG_LOGS=true
+CHAIN_ETH_ZKSYNC_NETWORK_ID=270
+CONTRACTS_ERA_CHAIN_ID="270"
+ETH_CLIENT_WEB3_URL= ""
+# MISC_ETHERSCAN_API_KEY = ""
```
Comment (StanislavBreadless) [inline etc/env/configs/stage.toml:6]:
```suggestion
MISC_ETHERSCAN_API_KEY = ""
```

**Review by StanislavBreadless — APPROVED** [review]

**Review by StanislavBreadless — APPROVED** [review]

**Pull request merged**